frontend/test/metabase/scenarios/onboarding/setup/setup.cy.spec.js

```diff
@@ -264,6 +265,6 @@ describeWithSnowplow("scenarios > setup", () => {
     cy.findByText("Welcome to Metabase");
     cy.button("Let's get started").click();
 
-    expectGoodSnowplowEvents(0);
```

Comment:
The 0->1 change doesn't really make sense to me, but I see it's in the original PR so I guess I won't question it